src/DetailsView/components/assessment-view-update-handler.ts

```diff
@@ -26,7 +26,7 @@ export interface AssessmentViewUpdateHandlerProps {
     deps: AssessmentViewUpdateHandlerDeps;
     assessmentNavState: AssessmentNavState;
     assessmentData: AssessmentData;
-    selectedRequirementIsEnabled: boolean;
+    isStepEnabled: boolean;
```

Comment:
Probably a bad merge here.

Reply:
This is intentional because isStepEnabled is the name of the property in TestStepView